packages/modules/viewer-react/src/components/app-ui/providers/DefaultContentGroupProvider.tsx

```diff
@@ -0,0 +1,75 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import {
+  ContentGroup,
+  ContentGroupProvider,
+  StandardContentLayouts,
+  UiFramework,
+} from "@itwin/appui-react";
+import { ViewportComponent } from "@itwin/imodel-components-react";
+import { viewWithUnifiedSelection } from "@itwin/presentation-components";
+import React from "react";
+
+import { getAndSetViewState } from "../../../services/iModel/index.js";
+import type {
+  BlankConnectionViewState,
+  ViewerViewCreator3dOptions,
+  ViewerViewportControlOptions,
+} from "../../../types.js";
+
+const UnifiedSelectionViewport = viewWithUnifiedSelection(ViewportComponent); // eslint-disable-line @typescript-eslint/no-deprecated
```

Comment:
The previous implementation of `DefaultContentGroupProvider` was choosing a viewport component based on whether the Viewer was supplied a selection storage or not.
- When supplied, we were using `ViewportComponent`.
- When not supplied, we were using `viewWithUnifiedSelection(ViewportComponent)`, with deprecated `viewWithUnifiedSelection` providing unified selection functionality to the viewport.

Now we're always using `viewWithUnifiedSelection(ViewportComponent)` - is this change intentional?

Reply:
Good catch. I switched it back to using just the `ViewportComponent`.
Personally, dont want to maintain both, and would prefer to drop deprecated behavior if possible. LMK your thoughts

Reply:
I attempted to internal the slectionStorage and cleaned up the code. Can you review my last commit and if theres additional things that need to be cleaned up can you call out where or tackle it? I know theres some leftover backwards compat stuff in packages/modules/viewer-react/src/hooks/useUnifiedSelectionScopes.tsx

Reply:
> Do you see value in exposing selectionStorage as a prop?

Yes. An app that uses `<Viewer />` will most likely have components that also want to hook into unified selection, and we need all such components to use the same selection storage ([example](https://github.com/iTwin/viewer-components-react/blob/2548a657148b0b052b28b1c3b838ba76b3e201fb/apps/test-viewer/src/UiProvidersConfig.tsx#L126)).

> any use cases why someone wouldnt use the default `createStorage()` from the `unified-selection` package?

No, I don't see why anyone would need that. But as explained above, it's about having access to the storage.

I think we should have the prop as required, or have it optional and have fallbacks to the deprecated `Presentation.selection`.